src/d3a/models/market/two_sided_pay_as_bid.py

```diff
@@ -73,6 +73,8 @@ def bid(self, price: float, energy: float, buyer: str, seller: str,
         if energy <= 0:
             raise InvalidBid()
 
+        if original_bid_price is None:
```

Comment:
Just for my understanding: This case is for the original bid, right?

Reply:
Exactly, this is the case where the bid is first placed in a market.